### ISSUE TITLE
Add a way to specify dependencies that are only in test_dependencies[] in .info files

### ISSUE THREAD

**Issue by fabianx-ai** [LionsAd/drupal_ti#9]
drafty specifies field_collection and entity_translation only as test_dependencies[], which drush does not support.

It also fails on drush/drush:master completely as it specifies drafty_enforce, but drush can't find it - even though this is in drafty.

**Comment by Decipher** [comment]
Maybe as an alternative to this, you could have a specific makefile for drupal_ti; drupal-ti.make?

My specific usecase is that one of the dependencies in my module has a bug in the stable that only occurs on install, it's resolved in dev, but that doesn't help me with testing my module. The workaround (I'm currently working on) is to force the download of the dev version after Drupal has been downloaded.

**Comment by LionsAd** [comment]
@Decipher Yes,that is what I do as well, see e.g. https://github.com/md-systems/pathauto/pull/51/files#diff-cac53111fcfaee8cad3f59282e74ab86R1 for a way to do this with the --script command.

**Comment by Decipher** [comment]
While it wouldn't necessarily help with the case you linked me to, #30 would simplify things for a non-specific dev release as you'd be able to run 'drush dl MODULE --dev -y' without first having to traverse to the specific directory.

**Comment by LionsAd** [comment]
Indeed.
